### DISCUSSION TITLE
CMU 10-414/714: Deep Learning Systems - csdiy.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#599] [Announcements]
# CMU 10-414/714: Deep Learning Systems - csdiy.wiki

CS自学指南

https://csdiy.wiki/en/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%E7%B3%BB%E7%BB%9F/CMU10-414/

**Comment by aixi** [comment]
大佬有最新的答案吗？某些测试就是跑不过，感觉很伤

**Reply by parthkhxndelwal** [reply]
該死的兄弟。你受傷了！

**Comment by FilipFan** [comment]
Based on my experience, the unique values of this course are:

- Examines neural networks at the mathematical computation level
- Covers core principles of NN frameworks, like Automatic Differentiation and Computational Graphs
- Explores the core software architecture of NN frameworks
- Provides hands-on practice with CUDA GPU acceleration programming
